project.json

```diff
@@ -25,7 +25,9 @@
       {"username": "vsoch", "aliases": ["github/vsoch", "discourse/vsoch"]},
       {"username": "amico", "aliases": ["github/amico-dj", "discourse/amico"]},
       {"username": "hammadj", "aliases": ["github/hammadj", "discourse/meta_dreamer"]},
-      {"username": "miyazono", "aliases": ["github/miyazono", "discourse/miyazono"]}
+      {"username": "miyazono", "aliases": ["github/miyazono", "discourse/miyazono"]},
+      {"username": "LB", "aliases": ["github/LexBirdie", "discourse/LB"]},
+      {"username": "Bex", "aliases": ["github/Bex2594", "discourse/Bex"]}
```

Comment:
There's 2 Bex's on the forums.
You're sure it's not https://discourse.sourcecred.io/u/bex2594
rather than https://discourse.sourcecred.io/u/bex?

Reply:
Good catch. I think those are both @Bex2594, I'll resolve to both of htem.